app/v1/models/response/package.scala

```diff
@@ -0,0 +1,31 @@
+/*
+ * Copyright 2019 HM Revenue & Customs
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package v1.models
+
+import org.json4s.CustomSerializer
+import org.json4s.JsonAST.JDecimal
+
+package object response {
+  object BigDecimalSerializer extends CustomSerializer[BigDecimal](_ =>
```

Comment:
@MarkAKelly BigDecimalSerializer was moved to a package object. It's still here.